MCprep_addon/world_tools.py

```diff
@@ -1182,7 +1188,7 @@ def cleanup_scene(self):
 		if self.open_folder:
 			bpy.ops.mcprep.openfolder(folder=self.filepath)
 
-	def create_panorama_cam(self, name, camera_data, rot, loc):
+	def create_panorama_cam(self, name: str, camera_data: Camera, rot: VectorType, loc: VectorType) -> Object:
```

Comment:
Again to prevent confusion
```suggestion
	def create_panorama_cam(self, name: str, camera_data: Camera, rot: VectorType, loc: VectorType) -> bpy.types.Object:
```